src/operation/find/mod.rs

```diff
@@ -21,7 +21,7 @@ use super::CursorResponse;
 pub(crate) struct Find<T> {
     ns: Namespace,
     filter: Option<Document>,
-    options: Option<FindOptions>,
+    options: Option<Box<FindOptions>>,
```

Comment:
`find_one` was one of the bigger ones and `FindOptions` itself is pretty big, so I opted to box it here too for some additional reduction in size.